docs/source/test_network.md

```diff
@@ -0,0 +1,578 @@
+   -ca <use CAs> -  create Certificate Authorities to generate the crypto material
+   -r <max retry> - CLI times out after certain number of attempts (defaults to 5)
+   -d <delay> - delay duration in seconds (defaults to 3)
+   -f <docker-compose-file> - specify which docker-compose file use (defaults to docker-compose-cli.yaml)
+   -s <dbtype> - the database backend to use: goleveldb (default) or couchdb
+   -l <language> - the programming language of the chaincode to deploy: go (default), javascript, or java
+   -v <version>  - chaincode version. Must be a round number, 1, 2, 3, etc
+   -i <imagetag> - the tag to be used to launch the network (defaults to "latest")
+   -verbose - verbose mode
+ network.sh -h (print this message)
+
+Example use
+
+Taking all defaults:
+ network.sh up
+
+Using flags:
+ network.sh up createChannel -ca -c mychannel -s couchdb -i 1.4.0
+ network.sh createChannel -c channelName
+ network.sh deployCC -l node
+```
+
+From inside the `test-network` directory, you can bring up the network by issuing
+the following command. You will experience problems if you try to run the script
+from another directory:
+```
+./network.sh up
+```
+
+This command creates a Fabric network that consists of two peer nodes, one
+ordering node, and an instance of the Fabric tools container. If the command
+completes successfully, you script will print out the following logs of the nodes
+being created.
+```
+
+Creating network "net_byfn" with the default driver
+Creating volume "net_orderer.example.com" with default driver
+Creating volume "net_peer0.org1.example.com" with default driver
+Creating volume "net_peer0.org2.example.com" with default driver
+Creating peer0.org1.example.com ... done
+Creating orderer.example.com    ... done
+Creating peer0.org2.example.com ... done
+Creating cli                    ... done
+CONTAINER ID        IMAGE                               COMMAND             CREATED             STATUS                  PORTS                    NAMES
+3d944d39fab3        hyperledger/fabric-tools:latest     "/bin/bash"         2 seconds ago       Up Less than a second                            cli
+280eaff2c840        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 1 second             0.0.0.0:9051->9051/tcp   peer0.org2.example.com
+0907470d42ee        hyperledger/fabric-orderer:latest   "orderer"           5 seconds ago       Up 1 second             0.0.0.0:7050->7050/tcp   orderer.example.com
+f1edf38980b5        hyperledger/fabric-peer:latest      "peer node start"   5 seconds ago       Up 2 seconds            0.0.0.0:7051->7051/tcp   peer0.org1.example.com
+```
+
+If you don't get this result, jump down to [Troubleshooting](#troubleshooting)
+for help on what might have gone wrong.
+
+### The components of the test network
+
+When the network running on your machine, you should take some time to examine
+what was created by the script. Run the following command to list all of Docker
```

Comment:
After your test network has deployed, take some time to examine the components inside of it.

Reply:
Done